packages/native/test/lib/ElementAssertion.test.tsx

```diff
@@ -1,13 +1,40 @@
 import { AssertionError, expect } from "@assertive-ts/core";
-import { render } from "@testing-library/react-native";
+import { fireEvent, render } from "@testing-library/react-native";
+import { useState, useCallback } from "react";
 import {
   View,
   TextInput,
   Text,
+  Modal,
+  Button,
 } from "react-native";
 
 import { ElementAssertion } from "../../src/lib/ElementAssertion";
 
+const SimpleToggleText: React.FC = () => {
+  const [isVisible, setIsVisible] = useState(true);
+
+  const handleToggle = useCallback((): void => {
+    setIsVisible((prev: boolean) => !prev);
+  }, []);
+
+  return (
+    <View>
+      <Text
+        testID="textElement"
```

Comment:
Using the testId should be our last resort to query elements. For instance, this element can be found using a `getByText("Toggle me!")` query.